chatterbot/ext/django_chatterbot/models.py

```diff
@@ -2,16 +2,34 @@
 
 
 class Statement(models.Model):
+    """A short (<255) chat message, tweet, forum post, etc"""
 
     text = models.CharField(
         unique=True,
         blank=False,
         null=False,
+        default='<empty>',
```

Comment:
Good point. Hadn't thought of that. I was just trying to get the amin interface to display something on the Statements list view so that the empty statements have a link that can be clicked to "walk the graph" of statement response. Will remove it.